mongo/integration/index_view_test.go

```diff
@@ -138,25 +139,55 @@ func TestIndexView(t *testing.T) {
 		})
 	})
 	mt.Run("create many", func(mt *mtest.T) {
-		iv := mt.Coll.Indexes()
-		indexNames, err := iv.CreateMany(mtest.Background, []mongo.IndexModel{
-			{
-				Keys: bson.D{{"foo", -1}},
-			},
-			{
-				Keys: bson.D{{"bar", 1}, {"baz", -1}},
-			},
-		})
-		assert.Nil(mt, err, "CreateMany error: %v", err)
-		assert.Equal(mt, 2, len(indexNames), "expected 2 index names, got %v", len(indexNames))
+		mt.Run("success", func(mt *mtest.T) {
+			iv := mt.Coll.Indexes()
+			indexNames, err := iv.CreateMany(mtest.Background, []mongo.IndexModel{
+				{
+					Keys: bson.D{{"foo", -1}},
+				},
+				{
+					Keys: bson.D{{"bar", 1}, {"baz", -1}},
+				},
+			})
+			assert.Nil(mt, err, "CreateMany error: %v", err)
+			assert.Equal(mt, 2, len(indexNames), "expected 2 index names, got %v", len(indexNames))
 
-		verifyIndexExists(mt, iv, index{
-			Key:  map[string]int{"foo": -1},
-			Name: indexNames[0],
+			verifyIndexExists(mt, iv, index{
+				Key:  map[string]int{"foo": -1},
+				Name: indexNames[0],
+			})
+			verifyIndexExists(mt, iv, index{
+				Key:  map[string]int{"bar": 1, "baz": -1},
+				Name: indexNames[1],
+			})
 		})
-		verifyIndexExists(mt, iv, index{
-			Key:  map[string]int{"bar": 1, "baz": -1},
-			Name: indexNames[1],
+		mt.Run("uses writeconcern", func(mt *mtest.T) {
```

Comment:
nit: This can be `mt.RunOpts` with the `CollectionOptions` option instead of using `CloneCollection` below.

Reply:
done